pronto_mia/lib/ui/views/login/login_view.dart

```diff
@@ -18,60 +19,41 @@ class LoginView extends StatelessWidget with $LoginView {
       viewModelBuilder: () => LoginViewModel(),
       onModelReady: (model) => listenToFormUpdated(model),
       builder: (context, model, child) => Scaffold(
-        body: Container(
-          padding: const EdgeInsets.all(16.0),
-          child: Column(
-            mainAxisAlignment: MainAxisAlignment.center,
-            children: [
-              Image.asset(
+        body: Column(
+          children: [
+            Container(
+              padding: const EdgeInsets.all(16.0),
+              child: Image.asset(
                 'assets/images/pronto_logo.png',
-                fit: BoxFit.contain,
+                height: 200,

```

Comment:
Does unfortunately not help with the pixel problem.

Reply:
I suggest that you have a look at this, since I can't reproduce the problem on my machine anymore.